docs/userguide/src/portingguide/topics/prefix.md

```diff
@@ -0,0 +1,5 @@
+# Special topics
+
+Every VM is special in some way.  Because of this, some VM bindings may use MMTk features not
```

Comment:
If we change the title, the paragraph needs to be changed accordingly. I feel it is more reasonable that this section is dedicated to how to implement different runtime features with MMTk.

Reply:
Yes. I changed this part, too.